perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/WrongReferenceAttributeValueException.java

```diff
@@ -89,6 +89,12 @@ public WrongReferenceAttributeValueException(AttributeDefinition attribute, Attr
 		this.attribute = attribute;
 		this.referenceAttribute = referenceAttribute;
 	}
+
+	public WrongReferenceAttributeValueException(AttributeDefinition attribute, Object attributeHolder, String message) {
+		super(attribute.toString() + " Set for: " + attributeHolder + " - " + message);
+		this.attribute = attribute;
+	}
```

Comment:
This is not correct way how to resolve WrongReferenceAttributeValueException. If you don't have reference, than it is weird. You can always use method and add NULL objects to the reference attribute and his holders if needed.

Reply:
Okay, the new constructor was removed.